hosts/testbed/network.nix

```diff
@@ -1,11 +1,27 @@
-{ hosts, ... }:
+{ pkgs, hosts, ... }:
 {
-  networking.interfaces."enp1s0" = {
+  environment.systemPackages = [
+    pkgs.hostapd
+  ];
+
+  netowrking.interfaces."enp1s0" = {
     ipv4.addresses = [
       {
-        address = hosts.testbed.ip;
+        address = "192.168.1.32";
```

Comment:
hardcoded ip address, use hsots.testbed.ip. also that's the panda's ip not testbed's